recipes/default.rb

```diff
@@ -33,6 +33,7 @@
 # Install required libraries
 #
 if node['platform'] == 'ubuntu'
+  package 'unzip'
```

Comment:
Why do you add this explicit package dependency? This is [transitively covered](https://github.com/chef-cookbooks/ark/blob/7559940dc37cfd672178d15571b7178f267178cb/attributes/default.rb#L32) by the `ark` cookbook.